patient_portal/messaging/messages.php

```diff
@@ -50,29 +50,32 @@
 foreach ( $result as $iter ) {
     $theresult[] = $iter;
 }
-$dashuser = Array();
+
+$dashuser = array();
 if(IS_DASHBOARD){
-    $dashuser = getUserIDInfo(IS_DASHBOARD);
+    $dashuser = getUserIDInfo($_SESSION['authUserID']);
 }
-function getAuthPortalUsers(){
+
+function getAuthPortalUsers()
+{
     $resultpd = $resultusers= $resultpatients = array ();
     if( IS_DASHBOARD ){ // admin can mail anyone
-        $resultusers[] = ['userid'=>'99998','username'=>'-- Auth Users --'];
-        $authusers = sqlStatement( "SELECT users.id as userid, CONCAT(users.fname,' ',users.lname) as username  FROM users WHERE authorized = 1" );
+        $authusers = sqlStatement("SELECT users.username as userid,
+        CONCAT(users.fname,' ',users.lname) as username, 'user' as type FROM users WHERE authorized = 1");
         while( $row = sqlFetchArray($authusers) ){
             $resultusers[] = $row;
         }
-        $resultpatients[] = ['userid'=>'99999','username'=>'-- Portal Patients --'];
-        $authpatients = sqlStatement( "SELECT patient_data.pid as userid, Concat_Ws(' ', patient_data.fname, patient_data.lname) as username FROM patient_data WHERE allow_patient_portal = 'YES'" );
+
+        $authpatients = sqlStatement("SELECT LOWER(CONCAT(patient_data.fname, patient_data.id)) as userid,
+        CONCAT(patient_data.fname,' ',patient_data.lname) as username,'p' as type,patient_data.pid as pid FROM patient_data WHERE allow_patient_portal = 'YES'");
         while( $row = sqlFetchArray($authpatients) ){
             $resultpatients[] = $row;
         }
         $resultpd[] = array_merge($resultusers,$resultpatients);
         return $resultpd[0];
-    }
-    else{
+    } else{
         $resultpd = array ();
-        $authusers = sqlStatement( "SELECT users.id as userid, CONCAT(users.fname,' ',users.lname) as username  FROM users WHERE authorized = 1" );
+        $authusers = sqlStatement("SELECT users.username as userid, CONCAT(users.fname,' ',users.lname) as username  FROM users WHERE authorized = 1");
```

Comment:
If using the user name,  should keep it username....just to pick...

Reply:
Will make that suggestion